ethereumj-core/src/main/java/org/ethereum/sharding/ShardingEntry.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) [2016] [ <ether.camp> ]
+ * This file is part of the ethereumJ library.
+ *
+ * The ethereumJ library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The ethereumJ library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with the ethereumJ library. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.ethereum.sharding;
+
+import org.apache.commons.lang3.StringUtils;
+import org.ethereum.cli.CLIInterface;
+import org.ethereum.config.SystemProperties;
+import org.ethereum.facade.Ethereum;
+import org.ethereum.facade.EthereumFactory;
+import org.ethereum.mine.Ethash;
+import org.ethereum.sharding.config.ShardingConfig;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * @author Mikhail Kalinin
+ * @since 26.07.2018
+ */
+public class ShardingEntry {
+
+    public static void main(String args[]) throws IOException, URISyntaxException {
+        System.setProperty("ethereumj.conf.res", "sharding.conf");
```

Comment:
btw I think it will be good feature to let user pass several configs
and this way we will not loose any user config already passed in such parameter

Reply:
What do you mean? That we _will_ loose user config passed this way?

Reply:
You can pass only one config with `ethereumj.conf.res`, so in such case if something is passed, it will be overwritten. It will be cool if we could pass several configs. Not for sharding specifically of course, generally.